.github/workflows/tests.yml

```
@@ -44,7 +44,7 @@ jobs:
 
     style:
         name: PHP Style Check
-        uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@main
+        uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@add-rules-for-cs-fixer
```

Change to main once https://github.com/GoogleCloudPlatform/php-tools/pull/152 is merged

```suggestion
        uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@main
```